deploy-cloudrun/action.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Deploy to Cloud Run
+author: GoogleCloudPlatform
+description: |-
+  Cloud Run is a fully managed compute platform that automatically scales your
+  stateless containers. Use this action to deploy a container that has already
+  been uploaded to the Container Registry repository to Cloud Run.
+
+inputs:
+  image:
+    description: |-
+      Name of the container image to deploy (e.g. gcr.io/cloudrun/hello:latest).
+      Required if not using a service YAML.
+    required: false
+
+  service:
+    description: |-
+      ID of the service or fully qualified identifier for the service.
+      Required if not using a service YAML.
+    required: false
+
+  region:
+    description: |-
+      Region in which the resource can be found.
+    required: false
+    default: us-central1
```

Comment:
Is this a conventional default region with other GCP services? Just want to make sure we're following established conventions vs defining our own.

Reply:
Yes per DevRel but not officially, I can remove and make this required?